.eslintrc-client-test

```diff
@@ -0,0 +1,7 @@
+---
+extends:
+  - "defaults/configurations/walmart/es6-test"
+  - "defaults/configurations/walmart/es6-react"
```

Comment:
@ryan-roemer Interesting – in that case we might want to add a `es6-react-test` to the set of Walmart defaults, so we can get the rules from both.